src/main/scala/com/lightbend/paradox/unidoc/UnidocKeys.scala

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.paradox.unidoc
+
+import sbt._
+
+object UnidocKeys {
+  val unidocRootPackage = settingKey[String]("")
```

Comment:
[`FastClasspathScanner`](https://github.com/lukehutch/fast-classpath-scanner/blob/master/src/main/java/io/github/lukehutch/fastclasspathscanner/FastClasspathScanner.java) requires the root package name, so you need to specify it.

Reply:
Sure, I can do that, but I think the repository name should also be changed. Could you do that, since I don't have entitlement to change the repo name?

Reply:
Sure. Repo renamed to `sbt-paradox-apidoc`

Reply:
Created a PR to extract the testkit project: https://github.com/lightbend/paradox/pull/211

Reply:
And here is a diff how it could be used in this plugin: https://github.com/2m/sbt-paradox-apidoc/commit/a61669b35e3b17cf547c07a1a8614bcc9a119b16